<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>tcp_serve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>hello</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>snake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>create_license</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
